spec/support/account_signup_helper_spec.rb

```diff
@@ -0,0 +1,22 @@
+module AccountSignupHelper
+  def self.test_ec_key_fixture
+    "-----BEGIN EC PRIVATE KEY-----\nMHcCAQEEIBjlMYhN/1FiZX7RRUyKpqDM2bPIHQswhjm+ZPaOXlgQoAoGCCqGSM49\nAwEHoUQDQgAE2pGXU3YkENblp+SMC8CWpkwPDdWXJdEsXoSQ8dWNYxFFbnXKN4Vv\nRGeLYoNKjuK/2t9rlEEumIzm5EtMxts7Jg==\n-----END EC PRIVATE KEY-----\n"
+  end
+
+  def account_feature_flag_environment_variables
+    {
+      "GOVUK_ACCOUNT_OAUTH_CLIENT_ID": "Application's OAuth client ID",
```

Comment:
I've pushed a branch of this without climate control and can see all of the tests passing locally: https://github.com/alphagov/finder-frontend/blob/brexit-save-results-without-climate-control/spec/features/brexit_checker/email_signup_spec.rb#L41-L45.

If possible, it would be good to avoid adding extra gems if we don't need to.

Reply:
Great, thanks for that I've amended that [here](https://github.com/alphagov/finder-frontend/pull/2207/commits/99c8dd96c269196f8ebe26fcdea4d090d3ca237c#diff-d1ee4be2d213856b5129c118619dea2aR20-R24) and was actually able to slim down a few more ENV VARS that weren't having any impact on the tests.